packages/devtools_app/lib/src/shared/console/widgets/description.dart

```diff
@@ -255,7 +250,7 @@ class DiagnosticsNodeDescription extends StatelessWidget {
     final defaultStyle = DefaultTextStyle.of(context).style;
     final baseStyle = style ?? defaultStyle;
     TextStyle textStyle = baseStyle.merge(
-      inspector_text_styles.textStyleForLevel(
```

Comment:
this seems a bit inverted. these are much more inspector specific styles than console styles. If you want a neutral name, these could be diagnostics_text_styles.

Reply:
Renamed and moved to shared root in #5021